hca/staging_area_validator.py

```diff
@@ -299,6 +299,7 @@ def validate_file_json(
             print(f"Validating JSON of {file_name}")
             try:
                 self.validator.validate_json(file_json, self.total_retries, schema)
+                # self.validator.validate_json(file_json, schema)
```

Comment:
@bahill can you remove this commented out line if it's not necessary?

Reply:
Missed one - rebuilding etc now. I'll push when I've got another build ready for PyPi